test/Test/BaseDAO/Proposal.hs

```diff
@@ -266,7 +266,9 @@ flushAcceptedProposals = uncapsNettest $ do
   key1 <- createSampleProposal 1 owner1 dao
   checkTokenBalance (DAO.frozenTokenId) dao owner1 10
 
-  callFrom (AddressResolved admin) dao (Call @"Set_voting_period") 20
+  -- Use 60s for voting period, since in real network by the time we call
+  -- vote entrypoint 30s is already passed.
+  callFrom (AddressResolved admin) dao (Call @"Set_voting_period") 60
   callFrom (AddressResolved admin) dao (Call @"Set_quorum_threshold") 1
```

Comment:
Hm, we make many calls between `propose` and `vote`. We call `set_voting_period` and `set_quorum_threshold`, also each `checkTokenBalance` makes 2 operations. Can we reorder operations here, so that `vote` is called` immediate after `propose` (or there is at least 1 call between them)?

Note that even if we do it, these tests still seem a bit fragile to me. AFAIU there are no strict guarantees about time between blocks. Sometimes bakers can miss their slots and there can be more than 30 seconds between 2 blocks in testnet. And in mainnet it's 1 minute on average. I doubt we'll ever test contracts in mainnet this way, but I still don't like that we rely on block interval. So:
1. I think we can more or less safely assume that there can be at most 1 minute between 2 blocks. Usually it's 30 seconds, but it's not guaranteed, so let's multiply it by 2. If we reorder actions so that `vote` is called right after `propose`, I think 1 minute should be fine as voting period. If we make `n` operations between `vote` and `propose`, we should have `n + 1` minutes as voting period.
2. Waiting for 1 minute is sad when we run this test in local chain with 1 second between blocks. I think we need to add a new method that returns an approximate time between blocks. I'll create an issue about that.
3. I think the only way to reliably write this test is to put `propose` and `vote` calls in one batch. However, batching is not merged in `cleveland` yet and there are some issues with it. So it won't be available sufficiently soon :disappointed: 

So I think that in order to run this test in real network we need some changes in `cleveland`. At least we need a way to know block interval because we don't want to wait for 1 minute in local chain tests. For this reason I propose to run these tests only in emulator for now. When we add at least block interval getter to `cleveland` (preferably also proper batching), we will restore this test in real network.

AFAIU the same applies to other tests fixed in this PR as well, so let's run them only on emulator, too.

Reply:
That makes sense, Ivan. I've reorder the operations, and make those tests only run in the emulator for now.